recipes/diligent-fx/all/BuildUtils.cmake

```diff
@@ -0,0 +1,399 @@
+if(PLATFORM_WIN32 OR PLATFORM_UNIVERSAL_WINDOWS)
```

Comment:
yes, the whole file is from diligent-core. And it is not allowed to package a *.cmake file.
Maybe it would be possible to include it from diligent-core source, but I don't know how

Reply:
Exactly the same question was raised, in diligent-tools PR and it was acceptable